src/components/Textarea/Textarea.tsx

```diff
@@ -1,44 +1,30 @@
 import React from 'react';
 import classNames from 'classnames';
-
+import { InputProps } from '../Field';
```

Comment:
Does it make sense to call this FieldProps instead of InputProps?

Reply:
Yeah, makes sense. Split these up into Field/Input